test/crs/conversions.jl

```diff
@@ -807,6 +865,18 @@
       @inferred convert(LatLon{WGS84{2296}}, c4)
       @inferred convert(LatLon{WGS84{0}}, c5)
     end
+
+    @testset "LatLonAlt: Datum conversion" begin
+      # TODO
+    end
+
+    @testset "GeocentricLatLon: Datum conversion" begin
+      # TODO
+    end
+
+    @testset "GeocentricLatLonAlt: Datum conversion" begin
+      # TODO
+    end
```

Comment:
I think this PR only adds Datum conversions for `GeocentricLatLon`. `La LonAlt` and `GeocentricLatLonAlt` do not yet have these conversions.

Reply:
If you want, you can add these methods in this RP.

Reply:
I would move them to the next PR, honestly, mostly because I would need to double-check the math by hand or use the output of a conversion as my test, which would probably make the test not particularly significative (like 1=1)